features/step_definitions/background_steps.rb

```diff
@@ -27,6 +27,12 @@
   end
 end
 
+Given("the following invoices exist") do |table|
+  table.hashes.each do |invoice_attrs|
+    create(:invoice, invoice_attrs.merge(user: @user))
```

Comment:
`user = User.find_by(email: company_attrs[:user]) || create(:user, email: company_attrs[:user])`
please use a solution like this for the user

Reply:
ok